stats/opentelemetry/opentelemetry.go

```diff
@@ -220,6 +231,10 @@ type metricsInfo struct {
 
 	startTime time.Time
 	method    string
+
+	labelsReceived bool
+	labels         map[string]string // labels to attach to metrics emitted
+	xDSLabels      map[string]string
```

Comment:
Nit: `xdsLabels`

Reply:
Done. I know the policy is acronym or ACRONYM based off first letter but I guess this felt undefined because lowercaseUPPERCASEUPPERCASE in the usual usage. I guess the first letter is what everything in acronym should be even if acronym is a mix in standard usage.